.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '23 23 * * 2'
```

Comment:
Why do you schedule it at that time instead of running it right away?

Reply:
Not really with a reason, this workflow was generated by Github so it comes from their template, but I was wondering the same to be honest

Reply:
I think we can leave, the idea is probably to check every x time the code, because in the mean time new security advisories can come out

Reply:
Agreed @V0XNIHILI